e2e/basics.spec.js

```diff
@@ -6,6 +6,8 @@ testUtil = require('../../heap/test/util');
 rnTestUtil = require('./rnTestUtilities');
 
 const doTestActions = async () => {
+  await element(by.id('Basics')).tap();
```

Comment:
This is to open the `Basics` tab but doesn't do any events we expect to record below?

Assuming so, might be helpful to comment to that effect.

Also, assuming so, will this start triggering something once react navigation capture is merged?

Reply:
Comments added.

Yeah, these will likely get autotracked along with any of the other ReactNavigation events.